app/controllers/calendar_controller.rb

```diff
@@ -23,7 +23,8 @@ def create_calendar_props
               days: calendar_days,
               times: calendar_times,
               unitEventLengthInSeconds: EventGrouper::GRANULARITY,
-              date: params[:date]}.compact
+              date: params[:date],
+              roomKinds: ConferenceRoom::KINDS}.compact
```

Comment:
I'll consider making it more unviersal in future - so that we can easily add different filters ;)